buildpipeline/pipelinejobs.groovy

```diff
@@ -16,12 +16,14 @@ def branch = GithubBranchName
 
 def linPipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/linux.groovy')
 def centos6Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/centos.6.groovy')
+def alpine36Pipeline = Pipeline.createPipelineForGithub(this, project, branch, 'buildpipeline/alpine.3.6.groovy')
```

Comment:
I think it would be nicer if we could find a way to parameterize the docker container instead of creating another groovy file. For both centos and alpine. We could even setup the container based on the Name property passed in.

Reply:
I would have done that if the groovy files were different only in the docker container name. But there is more difference - the RHEL one needs setting LD_LIBRARY_PATH for all the commands. Of course it can be expressed by a single file with more parametrization, but I'd prefer leaving that to someone with a better vision on what the shape should be than me.

Reply:
@weshaggard I'm not a huge fan of over-parameterizing this.  It's not just the docker container that changes in this case (also the test queues and sometimes other things).  Also the parameters currently end up in the job name, which makes the job names messy.

I'd rather have a few clean scripts with some duplicated logic than an overparameterized mess, like the official pipebuilds.

Reply:
I get the extra parameters can make it a little messy but we can use the parameters we already have and do conditions in the groovy files themselves. I really don't like having to maintain all these different scripts when the core logic is all the same.